packages/terra-clinical-header/tests/wdio/header-spec.js

```diff
@@ -0,0 +1,87 @@
+/* global browser, expect, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+
+describe('Clinical header', () => {
+  describe('Displays a default header with no content', () => {
+    before(() => {
+      browser.url('/#/raw/tests/clinical-header/default-header');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible();
+  });
+
+  describe('Displays a header with a title only', () => {
```

Comment:
Could we make these a bit less verbose?

Reply:
e349697495464cb6e3d98af1df2b6585576bfb49